test/src/single_line_comment/parser/highlight_single_line_comment_parser_test.dart

```diff
@@ -0,0 +1,311 @@
+import 'package:code_text_field/src/single_line_comments/parser/highlight_single_line_comment_parser.dart';
+import 'package:code_text_field/src/single_line_comments/parser/single_line_comments.dart';
+import 'package:code_text_field/src/single_line_comments/single_line_comment.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:highlight/highlight_core.dart';
+import 'package:highlight/languages/go.dart';
+import 'package:highlight/languages/java.dart';
+import 'package:highlight/languages/python.dart';
+import 'package:highlight/languages/scala.dart';
+
+void main() {
+  test('HighlightSingleLineCommentParser', () {
+    final data = [
```

Comment:
this array takes 250 lines. Would it be better to separate "data" into a separate file?

Reply:
1. It is no reused and will likely never by.
2. That would make the test case class public.
3. This array and the test it's in have many common reasons to change quite separate from other reasons affecting other files, so they are natural together.